src/framing/mod.rs

```diff
@@ -18,8 +18,8 @@ mod test_framing;
 
 #[derive(Debug, PartialEq, Clone)]
 pub struct MLSPlaintext {
-    pub(crate) group_id: GroupId,
-    pub(crate) epoch: GroupEpoch,
+    pub group_id: GroupId,
```

Comment:
Maybe `pub fn group_id(&self) -> GroupId` would be nicer?

Reply:
Yeah this was quick and dirty. I agree that having functions is nicer. I'll change that.